transformer/mappings_helpers.py

```diff
@@ -14,3 +14,21 @@ def closest_parent_value(self, uri, key):
             ancestor = self.aspace.client.get(a['ref']).json()
             if ancestor.get(key) not in ['', [], {}, None]:
                 return ancestor[key]
+
+    def closest_creators(self, uri):
+        """Iterates up through an archival object's ancestors looking for the
+        first value in a particular field. Returns that value."""
+        obj = self.aspace.client.get(uri).json()
+        for a in obj['ancestors']:
+            ancestor = self.aspace.client.get(a['ref']).json()
+            if len([c for c in ancestor.get("linked_agents") if c.get("role") == "creator"]):
+                creators = [c for c in ancestor.get("linked_agents") if c.get("role") == "creator"]
+                return creators
```

Comment:
I think we can combine this into a single line

Reply:
`return creators = [c for c in ancestor.get("linked_agents") if c.get("role") == "creator"]` ?

Reply:
`return [c for c in ancestor.get("linked_agents") if c.get("role") == "creator"]`

We don't need the variable assignment since we're just returning this value.

Reply:
Ah, ok, thanks. That makes sense.